src/javascript/Validation/validation.utils.js

```diff
@@ -35,38 +35,44 @@ export const onServerError = (error, formikActions, notificationContext, t, defa
     // Set submitting false
     formikActions.setSubmitting(false);
 
+    let notificationErrorMessage = t(defaultErrorMessage);
     const graphQLErrors = error.graphQLErrors;
     if (graphQLErrors && graphQLErrors.length > 0) {
         for (const graphQLError of graphQLErrors) {
             if (graphQLError.message && graphQLError.message.startsWith('javax.jcr.ItemExistsException')) {
                 // Custom handling for this error, system name is not valid
 
                 notificationContext.notify(t('content-editor:label.contentEditor.error.changeSystemName'), ['closeButton']);
+                notificationErrorMessage = null;
                 formikActions.setFieldError('ce:systemName', 'alreadyExist');
                 formikActions.setFieldTouched('ce:systemName', true, false);
-                return;
             }
 
-            if (graphQLError.message &&
-                graphQLError.message.startsWith('org.jahia.services.content.PropertyConstraintViolationException') &&
-                graphQLError.message.includes('Invalid link')) {
-                // Custom handling for invalid link error
+            if (graphQLError.errorType === 'GqlConstraintViolationException' &&
+                graphQLError.extensions && graphQLError.extensions.constraintViolations &&
+                graphQLError.extensions.constraintViolations.length > 0) {
+                // Constraint violation errors
 
-                const extractedErrorData = graphQLError.message.match(/^org.jahia.services.content.PropertyConstraintViolationException: (?<prop>(.*?)): Invalid link(?<link>(.*?))$/);
-                if (extractedErrorData && extractedErrorData.groups && extractedErrorData.groups.prop && extractedErrorData.groups.link) {
-                    const propNameIndex = extractedErrorData.groups.prop.lastIndexOf(' ');
-                    const propName = extractedErrorData.groups.prop.substring(propNameIndex + 1);
+                for (const constraintViolation of graphQLError.extensions.constraintViolations) {
+                    console.log('Constraint violation error: ', constraintViolation);
```

Comment:
is it necessary, maybe we can convert it to .error?

Reply:
CE do not support NodeConstraintViolation that are not attached to a prop.
That's why I prefer to log them to give details.
.error is fine but not mandatory, as it's mostly logged for informations but nothing is broken in the application, 
I prefer to use .error when and real error happen client side.